planning/behavior_velocity_planner/include/scene_module/occlusion_spot/manager.hpp

```diff
@@ -45,7 +45,7 @@ class OcclusionSpotModuleManager : public SceneModuleManagerInterface
   const char * getModuleName() override { return "occlusion_spot"; }
 
 private:
-  enum class ModuleID { PRIVATE, PUBLIC };
+  enum class ModuleID { DEFAULT, PUBLIC };
```

Comment:
Memo
default or general ?

Reply:
I change to GENERAL [cb73c4f](https://github.com/autowarefoundation/autoware.universe/pull/400/commits/cb73c4fce060c7a462180b6ed630a02fe41c8462)